src/BagnoDB/Serialization.fs

```diff
@@ -1,7 +1,11 @@
 namespace BagnoDB
 
     open MongoDB.Bson.Serialization
+    open MongoDB.Bson.Serialization.Serializers
 
     module Serialization =
         let bson bsonSerializer =
-            BsonSerializer.RegisterSerializationProvider bsonSerializer
\ No newline at end of file
+            BsonSerializer.RegisterSerializationProvider bsonSerializer
+
+        let registerDecimalSerializer () =
```

Comment:
I think we could create a `typed` RegisterSerializer function here like the `bson` function.

I mean something like that:
```fsharp
let bson bsonSerializer =
    BsonSerializer.RegisterSerializationProvider bsonSerializer

let typedBson<'TType> bsonSerializer =
    BsonSerializer.RegisterSerializer<'TType> bsonSerializer
```

Also, if we want to have also some specific serializers here (I don't have anything against). It would be better to name them like `let decimal`, `let something`, and add `[<RequiredQuilifiedAccess>]` above the `Serialization.

Reply:
fair enough. This is the only use case I know of atm and moved knowledge of Mongo specifics away from the caller.